clients/drcachesim/CMakeLists.txt

```diff
@@ -577,6 +577,12 @@ if (BUILD_TESTS)
         COMPILE_FLAGS "-Wno-attribute-alias")
     endif ()
 
+    add_executable(tool.drcacheoff.burst_reattach tests/burst_reattach.cpp)
+    configure_DynamoRIO_static(tool.drcacheoff.burst_reattach)
+    use_DynamoRIO_static_client(tool.drcacheoff.burst_reattach drmemtrace_static)
+    add_win32_flags(tool.drcacheoff.burst_reattach)
+    link_with_pthread(tool.drcacheoff.burst_reattach)
```

Comment:
I don't see any use of threads so this looks like a mistake?

Reply:
Good catch. I initially was trying to add more things in this test but I forgot to remove this eventually unused part.